test/use-swr-immutable.test.tsx

```diff
@@ -100,6 +100,36 @@ describe('useSWR - immutable', () => {
     await screen.findByText('data: 0')
   })
 
+  it('should do something', async () => {
+    async function fetcher(key: string) {
+      expect(key).toBe('constant')
+      return new Promise<string>(resolve => {
+        setTimeout(() => resolve('this is 1'), 1000)
+      })
+    }
+    preload('constant', fetcher)
+    function Component() {
+      const { data } = useSWRImmutable('constant', {
```

Comment:
If you change `useSWRImmutable` to `useSWR.` The test will pass, but semantically not what I want